### DISCUSSION TITLE
Row Indentation

### DISCUSSION THREAD

**Discussion by MentalGear** [revolist/revogrid#652] [General]
Within a tree structure, I would like to have the visual representation of indented rows, not just the text within the first cell to be indented with margin-right. Example:

<img width="666" alt="Screenshot 2021-12-12 at 12 01 01" src="https://user-images.githubusercontent.com/2837147/145709708-b754fda4-6093-4943-b028-fd25c3770e81.png">

Would that be possible with Revo?

**Comment by van-huyen** [comment]
Hi @MentalGear 

Did you mean tree view ? follow this:
https://revolist.github.io/revogrid/demo/jsx.tree.html
![image](https://user-images.githubusercontent.com/83264685/148475978-479c7a46-0ac1-45fe-a9f9-875ac8a335c0.png)

**Comment by bwl21** [comment]
<img width="1008" height="520" alt="2026-03-21 at 13 09 12_CleanShot_CleanShot@2x" src="https://github.com/user-attachments/assets/8177e5e8-cdb6-432f-a859-bc509f860e0b" />

i would like to have the indentation of the rows (test im Saal) according to the red line thus beeing aligned to the label of the group. As this id done by an inline style, it seems not to be possible using css.
